### PR TITLE
fix #3404

### LINKED ISSUE #3404: Teams doesn't start because of the apparmor profile
Hi,

I tried to start Teams (installed from MS's apt mirror) but it didn't work using the firecfg/firejail link.

$ teams
Reading profile /usr/local/etc/firejail/teams.profile
Reading profile /usr/local/etc/firejail/disable-devel.inc
Reading profile /usr/local/etc/firejail/disable-exec.inc
Reading profile /usr/local/etc/firejail/disable-interpreters.inc
Reading profile /usr/local/etc/firejail/whitelist-common.inc
Reading profile /usr/local/etc/firejail/whitelist-var-common.inc
Reading profile /usr/local/etc/firejail/electron.profile
Reading profile /usr/local/etc/firejail/disable-common.inc
Reading profile /usr/local/etc/firejail/disable-passwdmgr.inc
Reading profile /usr/local/etc/firejail/disable-programs.inc
Parent pid 82964, child pid 82965
Warning: cleaning all supplementary groups
Warning: cleaning all supplementary groups
Warning: cleaning all supplementary groups
Warning: not remounting /run/user/1001/gvfs
Warning: not remounting /run/user/1001/doc
Warning: cleaning all supplementary groups
Blacklist violations are logged to syslog
Child process initialized in 145.25 ms

Apparmor denial message was:
[ 2531.840022] audit: type=1400 audit(1588598895.892:135): apparmor="DENIED" operation="exec" profile="firejail-default" name="/usr/share/teams/teams" pid=81129 comm="nohup" requested_mask="x" denied_mask="x" fsuid=1001 ouid=0

aa-logprof suggested that "/usr/share/teams/teams mrix," should be added to the profile. I allowed it and Teams works now correctly however I don't know if it is a good modification.

Tanks.

### PR DESCRIPTION
**Pull request by rusty-snake**

### CODE REVIEW

File: etc/profile-m-z/teams.profile [hunk 1]
```diff
@@ -9,6 +9,8 @@ include teams.local
 # added by included profile
 #include globals.local
 
+# see #3404
+ignore apparmor
```
Comment (reinerh) [inline etc/profile-m-z/teams.profile:13]: there is a comment above about "ignore apparmor". can/should the comment be removed?

Comment (rusty-snake) [reply etc/profile-m-z/teams.profile:13]: You mean `# see #3404`? I added it for reference, we can remove it if it distract someone.

Comment (reinerh) [reply etc/profile-m-z/teams.profile:13]: no, i mean at the top of the file (line 5). :-)  Github didn't let me comment there.

Comment (rusty-snake) [reply etc/profile-m-z/teams.profile:13]: Uhh that what you mean. Yes it should be removed.

**Pull request merged**